### PR TITLE
[new-exec] add standalone executor test

### PR DESCRIPTION
**Pull request by zhiqiu**
### PR types

Others
### PR changes

Others
### Describe

[new-exec] add standalone executor test

### CODE REVIEW

**Review by wanghuancoder — DISMISSED** [review]
LGTM

File: paddle/fluid/framework/new_executor/CMakeLists.txt [hunk 1]
```diff
@@ -12,4 +12,18 @@ cc_library(stream_analyzer SRCS stream_analyzer.cc DEPS ${DEVICE_EVENT_LIBS} glo
...
 cc_library(standalone_executor SRCS standalone_executor.cc DEPS interpretercore)
 cc_test(workqueue_test SRCS workqueue_test.cc DEPS workqueue)
-# cc_binary(standalone_executor_test SRCS standalone_executor_test.cc DEPS interpretercore standalone_executor operator op_registry executor ${GLOB_OP_LIB} ${GLOB_OPERATOR_DEPS} profiler)
+
+if (WITH_TESTING)
+    add_custom_target(
+        download_program
+        COMMAND wget -nc https://paddle-ci.gz.bcebos.com/new_exec/lm_main_program 
+        COMMAND wget -nc https://paddle-ci.gz.bcebos.com/new_exec/lm_startup_program 
+    )
```
Comment (Aurelius84) [inline paddle/fluid/framework/new_executor/CMakeLists.txt:23]: There is a risky problem: If ops in framework are modified, the program maybe become unavailable.

Comment (zhiqiu) [reply paddle/fluid/framework/new_executor/CMakeLists.txt:23]: thx, I will update the `program` if needed.

**Review by XieYunshen — APPROVED** [review]
LGTM for `set_tests_properties(standalone_executor_test PROPERTIES TIMEOUT 100)`

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Pull request merged**